src/content/reference/manifest.mdx

```diff
@@ -560,6 +560,47 @@ selector:
   app.kubernetes.io/name: vote
 ```
 
+### services ([object], optional)
+
+A list of other containers (other than the main development container) that you want to put on developer mode.
+The difference with the main development container and the ones defined in this section is that the main development container gives you an interactive session, where you can, for example, execute a shell.
+The containers defined in this section run in detached mode, but they can also mount your local folders in a particular path.
```

Comment:
I don't think we should say "detached mode" unless we have a definition for that somewhere. I think it's better to remove this paragraph.